data/test/schema/Customer.isl

```diff
@@ -8,6 +11,7 @@ type::{
     firstName: { type: string, occurs: required },
     middleName: string,
     lastName: { type: string, codepoint_length: range::[min, 7], occurs: required },
+    age: { type: positive_int, occurs: optional },
```

Comment:
`occurs: optional` is the default for a field.  Please change this line to `age: positive_int,`